io/plugins/eu.esdihumboldt.hale.io.jdbc/src/eu/esdihumboldt/hale/io/jdbc/extension/DriverConfigurationExtension.java

```diff
@@ -63,7 +63,9 @@ protected String getIdAttributeName() {
 	@Override
 	protected DriverConfiguration create(String elementId, IConfigurationElement element) {
 		if (element.getName().equals("driver")) {
-			return new DriverConfiguration(elementId, element);
+			DriverConfiguration driverConfiguration = new DriverConfiguration(elementId, element);
+			if (!driverConfiguration.isFileBased())
```

Comment:
The DriverConfiguration fullfills more purposes than just the Driver listing in the UI. Wouldn't it make more sense to just hide file based drivers in the UI?